lnclient/ldk/ldk.go

```diff
@@ -266,6 +266,15 @@ func NewLDKService(ctx context.Context, cfg config.Config, eventPublisher events
 			case <-time.After(MIN_SYNC_INTERVAL):
 				ls.syncing = true
 
+				channels := ls.node.ListChannels()
+				for _, channel := range channels {
+					if channel.Confirmations != nil && channel.ConfirmationsRequired != nil && *channel.Confirmations < *channel.ConfirmationsRequired {
+						logger.Logger.WithField("channel_id", channel.UserChannelId).Debug("Using short sync time while opening channel")
+						ls.lastWalletSyncRequest = time.Now()
+						break
```

Comment:
why break?

Reply:
To exit the for loop if we already found a pending channel as there is no need to iterate the rest of the channels. It's not really necessary I guess. Do you think it should be removed?